network-api/networkapi/templates/fragments/continuous_scrolling_next.html

```diff
@@ -0,0 +1,11 @@
+{% load i18n %}
+<div class="cont-scrolling tw-w-full tw-h-screen tw-bg-blue-lightest tw-sticky tw-z-[2000] tw-overflow-hidden" data-href="{{ page.next_page.url }}">
+  <div class="tw-container tw-py-6">
+    <div class="tw-flex tw-flex-col tw-justify-center tw-items-center tw-text-black" >
+      <div class="tw-font-sans tw-text-xs tw-uppercase tw-mb-3 tw-tracking-wide">{% trans "Keep Scrolling For" %}</div>
```

Comment:
I’d just add some additional context for translators, as it’s not super clear it’s concatenated with the next page title. Ideally, we would move the title var inside the string, but I reckon it would quite complicate things here

```suggestion
      <div class="tw-font-sans tw-text-xs tw-uppercase tw-mb-3 tw-tracking-wide">{% trans "Keep Scrolling For" context "String followed by the next page title the user will reach by scrolling down" %}</div>
```